src/build_system.mli

```diff
@@ -61,3 +61,7 @@ val build_rules
   -> ?recursive:bool (* default false *)
   -> Path.t list
   -> Rule.t list Future.t
+
+val load_trace
+  :  unit
+  -> (Path.t, string) Hashtbl.t option
```

Comment:
The hash table doesn't make much sense outside of this module. We should just return the list of keys:

```
val all_targets_ever_built : unit -> string list
```